EZP-29827: Harmonized Preview Switcher in Content Section Editing view

inakijv opened the pull request:
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-29827](https://jira.ez.no/browse/EZP-29827)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)
<!-- Keep in mind: Your contribution has to be compatible with GPL-2.0 as well: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLModuleLicense -->


#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [X] Ready for Code Review

![ez platform 70](https://user-images.githubusercontent.com/9256718/48864233-f28e6e80-ed99-11e8-9c6c-067fb7645d89.png)
![ez platform 71](https://user-images.githubusercontent.com/9256718/48864234-f28e6e80-ed99-11e8-894a-3745df29af34.png)
![ez platform 72](https://user-images.githubusercontent.com/9256718/48864235-f3270500-ed99-11e8-9366-ebe0174490ac.png)

sunpietro commented on src/bundle/Resources/public/scss/_preview.scss:
```diff
@@ -54,14 +56,25 @@
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                flex: 0 0 34px;
```
Maybe `flex: 0 0 calculateRem(34px)`?

sunpietro commented on src/bundle/Resources/public/scss/_preview.scss:
```diff
@@ -76,32 +89,31 @@
             background: url('/bundles/ezplatformadminui/img/preview-tablet.png') top center no-repeat;
-            height: 1250px;
-            margin-top: 1rem;
+            height: calculateRem(1250px);
```
I'm wondering, do we need such big value for `height`? I mean, it will cause the scroll.

inakijv replied:
Hi @sunpietro, I'd say yes if we want to show the full `preview-tablet.png` img which is preview's objective. In any case given that you are pointing this out `preview-tablet.png` height is 1287px, hence maybe the correct height would be 1290px.

inakijv commented:
Updated based on review, and added fixup for responsiveness in order to avoid behavior of an enlarged switcher when resizing browser window (see screenshot of behavior detected and corrected)
![screen shot 2018-11-26 at 11 07 56 am](https://user-images.githubusercontent.com/9256718/49028777-30600e00-f171-11e8-8c31-5b49d16cf50d.png)

inakijv commented:
Ping @sunpietro
